client/README.md

```diff
@@ -119,16 +119,21 @@ NOTE: You can specify a RetentionPolicy as part of the batch points. If not
 provided InfluxDB will use the database _default_ retention policy.
 
 ```go
+
 func writePoints(clnt client.Client) {
 	sampleSize := 1000
-	rand.Seed(42)
 
-	bp, _ := client.NewBatchPoints(client.BatchPointsConfig{
+	bp, err := client.NewBatchPoints(client.BatchPointsConfig{
 		Database:  "systemstats",
 		Precision: "us",
 	})
+	if err != nil {
+		log.Fatal(err)
+	}
 
 	for i := 0; i < sampleSize; i++ {
+		rand.Seed(uint64(i))
```

Comment:
It is a good catch. I wanted to give a true randomness here by incrementing the seed with iterator. Every time this function runs, will create a new graph. But the mistake I made here is to get confused about how to do it, I think taking the seed with the way of the last commit will change the sequence in every run.